src/checks/json.check.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 const status = require('./status');
 const Check = require('./check');
+const fetch = require('node-fetch');
```

Comment:
for that reason, it may be better to do this in the test file instead, for now.
`global.fetch = require('node-fetch');`